applications/StructuralMechanicsApplication/tests/test_patch_test_large_strain.py

```diff
@@ -70,9 +76,9 @@ def _define_movement(self,dim):
             #define the applied motion - the idea is that the displacement is defined as u = A*xnode + b
             #so that the displcement is linear and the exact F = I + A
             A = KratosMultiphysics.Matrix(3,3)
-            A[0,0] = 0.1;     A[0,1] = 0.2;    A[0,2] = 0.0
-            A[1,0] = 0.05;    A[1,1] = 0.07;   A[1,2] = 0.0
-            A[2,1] = 0.0;     A[2,1] = 0.0;    A[2,2] = 0.0
+            A[0,0] =   0.10;  A[0,1] = 0.12; A[0,2] = 0.0
```

Comment:
ok with the change, but may i ask you why this decision?

Reply:
Because it was giving me problems (the jacobian appeared as inverted in first iteration)

Reply:
mmmmm

can u check again after the latest corrections?

Reply:
I can

Reply:
Nope, doesn't work

Reply:
Can you check if it works with the TL? If it doesn't keep it as it is

Reply:
It works

Reply:
... if it works with the TL and not with the UL, it means that the UL still has a problem. no?

Reply:
The thing is that the deformation impossed previously inverted one element, this in TL is not a problem because the integration is done in the reference confguration, but not for the Ul. I did many tests, the TL and UL give exactly the same solution

Reply:
if you use an hyperelastic law, with linear elastic that's not true (except for very small deformations)